conda_pack/tests/test_core.py

```diff
@@ -90,7 +87,6 @@ def test_errors_pip_overwrites():
     msg = str(exc.value)
     assert "pip" in msg
     assert "toolz" in msg
```

Comment:
I removed cytoolz because it requires the compiler suite on Windows to be available to run the test suite. That wasn't the case for me, but maybe we still want to test that?

Reply:
Fine by me.